Makefile

```diff
@@ -5,20 +5,17 @@
 default: test
 
 test:
-	python -m unittest discover
-
-coverage:
-	coverage run -m unittest discover
-	coverage report -m
+	tox -e py27,py34
```

Comment:
Pylint 1.4.4 doesn't support Python 3.5, and Pylint 1.5 crashes on edx-platform.